HW_7/Ex_34.py

```diff
@@ -0,0 +1,38 @@
+'''Задача 34:
+Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг от друга пробелами. Стихотворение Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+Ввод:                                       Вывод:
+пара-ра-рам рам-пам-папам па-ра-па-дам      Парам пам-пам
+'''
+
+# ___________________РЕШЕНИЕ________________________
+
+
+def rythm_check(request):
+
+    request_list = list(map(str, request.split(' ')))
```

Comment:
Андрей, метод `split` и так возвращает список строк. Всегда. Поэтому тут половина лишнего. И, если хотите разделить по пробелам, то лучше ничего не предавать методу `split`
```suggestion
    request_list = request.split()
```